18F/identity-idp
# LG-6436: Add hybrid document capture opt-in for in-person proofing

**Why:** So that anyone who is unsuccessful with document capture has an option to navigate to the in-person proofing flow, even if they're doing document capture through the "hybrid" experience.

**Testing Instructions:**

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing flow up to "How would you like to upload your state-issued ID?"
5. Click "Use your phone"
6. Enter a phone number and click "Continue"
7. In a private/incognito browser session, navigate to http://localhost:3000/test/telephony
8. Copy the URL from the first message and navigate to it in the private/incognito session
9. Use an [attention result YAML test document](https://developers.login.gov/testing/#document-upload) for the fields
10. Click Submit
11. Click "Verify your ID in person"
12. Select a location and continue
13. Click Continue
14. Observe that you are prompted to switch back to your computer
15. Observe that in the initial browser selection, you are redirected to the in-person flow 

**Screen Recording:**

https://user-images.githubusercontent.com/1779930/182161076-3ef1cb2a-f8de-4d0a-a5a9-1e8a19bc1ab8.mov
